adder/ipfsadd/add.go

```diff
@@ -390,9 +389,9 @@ func outputDagnode(out chan *api.AddedOutput, name string, dn ipld.Node) error {
 	}
 
 	out <- &api.AddedOutput{
-		Hash: dn.Cid().String(),
+		Cid:  dn.Cid().String(),
```

Comment:
I suspect this change will require update to js-ipfs-api, which still expects `Hash` in [file-result-stream-converter.js#L32](https://github.com/ipfs/js-ipfs-api/blob/4191525f69433e16dd09b862821b7f8b95f5ef06/src/utils/file-result-stream-converter.js#L32) and [L38](https://github.com/ipfs/js-ipfs-api/blob/4191525f69433e16dd09b862821b7f8b95f5ef06/src/utils/file-result-stream-converter.js#L38)
(It should try `Hash` and if missing use `Cid`)

Reply:
hey, this only affects the Cluster API /add. The IPFS Proxy /add output will be fully compatible with the ipfs output format.

Reply:
swap is here: https://github.com/ipfs/ipfs-cluster/pull/569/files#diff-66308423c5638970cde2dd02574201c3R431

Reply:
Ah, I get it now, thanks for explanation!